packages/core/src/browser/shell/tab-bars.spec.ts

```diff
@@ -0,0 +1,53 @@
+/********************************************************************************
+ * Copyright (C) 2019 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { enableJSDOM } from '../test/jsdom';
+enableJSDOM();
+import { expect } from 'chai';
+
+import { Title, Widget } from '@phosphor/widgets';
+import { TabBarRenderer } from './tab-bars';
+
+describe('tab-bar', () => {
+    describe('tab-name-disambiguation', () => {
```

Comment:
Can we follow the pattern present in other tests?
Namely, 

```ts
it('should...')
```

Example:
https://github.com/theia-ide/theia/blob/17bfcb2eee69acccade5106de050576f6c46206a/packages/core/src/common/command.spec.ts#L30-L36

Reply:
Sure!